src/app/components/wrapper-components/GameWrapper.ts

```diff
@@ -63,17 +63,20 @@ export abstract class GameWrapper<P extends Comparable> extends BaseWrapperCompo
     /**
      * This method is to be called only after view init.
      * It will create the game component and initialize its node.
-     * It returns true if succesful, or false if this is not a valid game.
+     * It returns true if successful, or false if this is not a valid game.
      */
     protected async createMatchingGameComponent(): Promise<boolean> {
         const componentType: MGPOptional<Type<AbstractGameComponent>> =
             await this.getMatchingComponentAndNavigateOutIfAbsent();
         if (componentType.isPresent()) {
+            // This waits for the config to be chosen
             const config: MGPOptional<RulesConfig> = await this.getConfig();
             await this.createGameComponent(componentType.get());
             this.gameComponent.config = config;
             this.gameComponent.node = this.gameComponent.rules.getInitialNode(config);
+            await this.setRole(this.role);
             await this.gameComponent.updateBoard(false);
+            this.gameComponent.redraw();
```

Comment:
1. mettre this.gameComponent.updateBoard(false) dans redraw ?
2. un meilleurs nom pour redraw ?

Reply:
Voilà, maintenant il y a updateBoardAndRedraw et showLastMoveAndRedraw !